main.py

```diff
@@ -1 +1,24 @@
 #Mad Libs Generator Project
+
+#Loop back to this point once code finishes
+loop = 1
```

Comment:
Great use of the while loop! Try starting the loop at 0 through the range of the number of iterations you want- remembering that the number you use for the loop is exclusive, meaning it is not included in the final count

Reply:
I agree!

Reply:
+1